extensions/interactions/base.py

```diff
@@ -111,6 +111,10 @@ class BaseInteraction(object):
     default_outcome_heading = None
     # Whether the solution feature supports this interaction.
     can_have_solution = None
+    # Whether to show a Submit button in the progress navigation area when the
+    # window width is narrow. The interaction JS must handle the
+    # EVENT_PROGRESS_NAV_SUBMITTED event broadcast by the nav Submit button.
+    show_nav_submit_button = False
```

Comment:
Remember to update the backend tests that validate interactions.

Reply:
Thanks for the tip!